### PR TITLE
Added deployment Steps on Cloudways

### PR DESCRIPTION
**Pull request by shahroznawaz**
Added the steps of launching Symfony applications on Cloudways Cloud Servers.

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
👍

File: deployment/cloudways.rst [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+   single: Deployment; Deploying to Cloudways Cloud Servers
+
+Deploying to Cloudways 
+======================
+
+Cloudways is a managed hosting provider for PHP based applications. This step by
+step article describes how to install Symfony on Cloudways servers and how to
+deploy Symfony applications using Git and SSH.
+
+Step 1: Prepare your server and application
```
Comment (xabbuh) [inline deployment/cloudways.rst]: Prepare Your Server and Application

Comment (xabbuh) [inline deployment/cloudways.rst]: Should we really add the step numbers in the headlines? This will require a lot of work when steps are not necessary any more in the future or new steps are to be added.

File: deployment/cloudways.rst [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+-------------------------------------------
+
+`Sign up`_ or login to your **Cloudways account** and create a new server by
+selecting PHP Stack from the **Select your Application** drop-down. Also, name
+your application, server and project accordingly.
+
+.. image:: /_images/deployment/cloudways/select-application.png
+   :alt: Select PHP Stack in Drop down
+
+Step 2: Select Cloud Provider
```
Comment (xabbuh) [inline deployment/cloudways.rst]: Select a Cloud Provider

File: deployment/cloudways.rst [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+-----------------------------
+
+Now select your favorite cloud provider. You can select any one from the five
```
Comment (xabbuh) [inline deployment/cloudways.rst]: should we just write "Select your favorite cloud provider from the list of available providers."? This way we won't have to update the article every time a new provider gets added.

File: deployment/cloudways.rst [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+-----------------------
+
+Once the server is completely launched, set the server timezone, because Symfony
+requires to define a valid timezone for PHP. Under **Server Management**, go to
+**Settings & Packages.**
+
+.. image:: /_images/deployment/cloudways/server-management.png
+   :alt: server management tab
+
+In **Basic** tab you can see **PHP Timezone.** Change the server timezone to the
```
Comment (xabbuh) [inline deployment/cloudways.rst]: In the [...]

File: deployment/cloudways.rst [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+appropriate value according to your location.
+
+.. image:: /_images/deployment/cloudways/server-timezone-1.png
+   :alt: Select server timezone
+
+Step 5: SSH Terminal
+--------------------
+
+After changing server timezone, in **Server Management** click on **Launch SSH
```
Comment (xabbuh) [inline deployment/cloudways.rst]: [...] changing the server [...]

File: deployment/cloudways.rst [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+.. image:: /_images/deployment/cloudways/master-credentials.png
+   :alt: Cloudways Master Credentials
+
+Login to SSH with your Master Credentials.
+
+.. image:: /_images/deployment/cloudways/symfony-7.png
+   :alt: login to SSH
+
+Since the application folder contains a ``index.php`` fie by default, remove it
```
Comment (xabbuh) [inline deployment/cloudways.rst]: [...] an `index.php` file [...]

File: deployment/cloudways.rst [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+--------------------------
+
+After a successful installation, in **Application Management** go to **Access
+detail** and copy your Application URL.
+
+.. image:: /_images/deployment/cloudways/mysql-access.png
+   :alt: database information
+
+Open your favorite web browser and access to the "Symfony Welcome page" using
+the following URL: http://your_application_url/your_project_directory/web
```
Comment (xabbuh) [inline deployment/cloudways.rst]: It looks wrong to me that you store your entire project in the publicly accessible document root. Is this really the way to go with Cloudways?

File: deployment/cloudways.rst [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+
+.. image:: /_images/deployment/cloudways/symfony-home.png
+   :alt: symfony homepage
+
+Finally your Symfony project is ready for development. 
+
+Deploying From Github 
+======================
+
+You can also deploy Symfony application from github. copy the HTTP URL of the repository,
```
Comment (xabbuh) [inline deployment/cloudways.rst]: applications

Comment (xabbuh) [inline deployment/cloudways.rst]: GitHub

Comment (xabbuh) [inline deployment/cloudways.rst]: Copy

File: deployment/cloudways.rst [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+
+.. image:: /_images/deployment/cloudways/http_url.png
+   :alt: symfony homepage
+
+Open SSH terminal on cloudways Using Master Credentials as shown in Step 5. Move to your project folder,
```
Comment (xabbuh) [inline deployment/cloudways.rst]: Cloudways

Comment (xabbuh) [inline deployment/cloudways.rst]: Open an SSH terminal [...]

File: deployment/cloudways.rst [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+`cd application/{your app folder}/public_html`
```
Comment (xabbuh) [inline deployment/cloudways.rst]: I would reword the sentence to "[...] Move to your project folder and run the clone command there:" and then show both commands in a `terminal` code block:

```rst
[...] Move to your project folder and run the clone command there:

.. code-block:: terminal

    cd application/{your app folder}/public_html
    git clone https://github.com/username/your-repository.git
```

File: deployment/cloudways.rst [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+
+Run the clone command here:
+
+`git clone https://github.com/username/your-repository.git`
+
+After successfull deployment change your database credentials in **parameters.yml**. you can find your db credentials
```
Comment (xabbuh) [inline deployment/cloudways.rst]: After a successful deployment [...]

Comment (xabbuh) [inline deployment/cloudways.rst]: You

File: deployment/cloudways.rst [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+in **Access details** as stated in step 7.
+
+
```
Comment (xabbuh) [inline deployment/cloudways.rst]: one blank line should be removed

File: deployment/cloudways.rst [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+.. _`Sign up`: https://platform.cloudways.com/signup
+
```
Comment (xabbuh) [inline deployment/cloudways.rst]: this blank line should be removed (a trailing newline is enough)

File: deployment/cloudways.rst (hunk 3 above)
Comment (shahroznawaz) [inline deployment/cloudways.rst]: @xabbuh there is no plan to add more providers yet. so I am changing the line as you say.

File: deployment/cloudways.rst (hunk 7 above)
Comment (shahroznawaz) [inline deployment/cloudways.rst]: Yeah it should be http://your_application_url/web. i am updating it

**Comment by shahroznawaz** [comment]
@xabbuh I have updated the steps please review it.

File: deployment/cloudways.rst (hunk 7 above)
Comment (xabbuh) [inline deployment/cloudways.rst]: But that would still mean that all your project files will be publicly available.

Comment (shahroznawaz) [inline deployment/cloudways.rst]: @xabbuh No only Web directory is Publicly accessible. Rest are Private.

Comment (xabbuh) [inline deployment/cloudways.rst]: Then I don't understand why I should append the `/web` path.

**Comment by shahroznawaz** [comment]
@javiereguiluz @xabbuh What's the status of this pull request?

**Comment by javiereguiluz** [comment]
@shahroznawaz I'm really sorry but we must close this pull request without merging it. Why? Because as part of the review of Symfony Docs for the upcoming Symfony 4 version, we've decided in #8721 to remove all third-party deployment articles. We've removed Heroku, Platform.sh, etc. So we cannot add new articles about this. Instead of this, we now link to the articles about deploying Symfony apps published by third-party services. Thanks for understanding it.
